test/relations.test.js

```diff
@@ -90,6 +91,18 @@ describe('relations', function () {
         });
       });
 
+      it('should create record on scope with promises', function (done) {
+        Book.create()
+        .then (function (book) {
+          return book.chapters.create()
+          .then (function (c) {
+            should.exist(c);
+            c.bookId.should.equal(book.id);
+            done();
+          });
+        }).catch(done);
```

Comment:
Coding style: place `.catch` on a new line. Please apply the fix everywhere in the test file.